tests/aws/services/lambda_/test_lambda.snapshot.json

```diff
@@ -3456,15 +3456,15 @@
     }
   },
   "tests/aws/services/lambda_/test_lambda.py::TestLambdaBaseFeatures::test_assume_role[1]": {
-    "recorded-date": "10-10-2023, 14:55:53",
+    "recorded-date": "07-11-2023, 17:06:55",
     "recorded-content": {
-      "invoke-result-assumed-role-arn": "arn:aws:sts::111111111111:assumed-role/lambda-autogenerated-<lambda-autogenerated-role-prefix>/<resource:1>"
+      "invoke-result-assumed-role-arn": "arn:aws:sts::869636330914:assumed-role/lambda-autogenerated-<lambda-autogenerated-role-prefix>/<function-name>@lambda_function"
```

Comment:
I guess we still need a transformer for the account here after clearing the default transformers 🤔